quickwit/Cargo.toml

```diff
@@ -88,6 +88,8 @@ indicatif = "0.17.3"
 itertools = "0.10.5"
 json_comments = "0.2"
 libz-sys = "1.1.8"
+lindera-core = "0.25.0"
```

Comment:
can you put all that behind a feature flag?

(this is a very independent feature and I don't want it to impact our day to day dev compilation time.)